apps/aecore/src/aec_peer_connection.erl

```diff
@@ -957,7 +957,7 @@ handle_new_micro_block(S, Msg) ->
                 %% in the full micro block case - conductor will do the necessary checks
                 aec_conductor:post_block(MicroBlock);
             true ->
-                {ok, #{ header := Header, tx_hashes := TxHashes }} =
+                {ok, #{ header := Header, tx_hashes := TxHashes, pof := PoF }} =
                     deserialize_light_micro_block(maps:get(micro_block, Msg)),
                 %% Before assembling the block, check if it is known, and valid
                 case pre_assembly_check(Header) of
```

Comment:
These checks are done for light micro block and not full micro block because additional state is kept in node while filling in the missing pieces of light micro block. And the checks listed there reflect this. So PoF addition does not require additional validation in pre_assembly_check

Ok.